.github/workflows/ci.yml

```diff
@@ -0,0 +1,94 @@
+---
+# Managed by modulesync - DO NOT EDIT
+# https://voxpupuli.org/docs/updating-files-managed-with-modulesync/
+
+name: CI
+
+on: pull_request
+
+concurrency:
+  group: ${{ github.head_ref }}
+  cancel-in-progress: true
+
+jobs:
+  setup_matrix:
+    name: 'Setup Test Matrix'
```

Comment:
Please update to 5.2.0, though you may need to wait for https://github.com/voxpupuli/puppet-augeasproviders_apache/pull/11#discussion_r872205680 to be resolved.

Reply:
I added a modulesync 5.2.0 commit but left `ci.yml` at the `5.1.0` version.

Reply:
I opened https://github.com/voxpupuli/modulesync_config/pull/780 which should resolve it. I'd prefer to have the 5.2.0 config, just like @bastelfreak did in https://github.com/voxpupuli/puppet-augeasproviders_apache/pull/11 since it's much shorter.